rust/agents/relayer/src/msg/serial_submitter.rs

```diff
@@ -256,92 +256,88 @@ impl SerialSubmitter {
+        else {
+            return Ok(false)
+        };
+
+        // Estimate transaction costs for the process call. If there are issues, it's likely
+        // that gas estimation has failed because the message is reverting. This is defined behavior,
+        // so we just log the error and move onto the next tick.
+        let tx_cost_estimate = match self
+            .mailbox
+            .process_estimate_costs(&msg.message, &metadata)
+            .await
         {
-            // Estimate transaction costs for the process call. If there are issues, it's likely
-            // that gas estimation has failed because the message is reverting. This is defined behavior,
-            // so we just log the error and move onto the next tick.
-            let tx_cost_estimate = match self
-                .mailbox
-                .process_estimate_costs(&msg.message, &metadata)
-                .await
-            {
-                Ok(tx_cost_estimate) => tx_cost_estimate,
-                Err(err) => {
-                    info!(msg=?msg, error=?err, "Error estimating process costs");
-                    return Ok(false);
-                }
-            };
-
-            // If the gas payment requirement hasn't been met, move to the next tick.
-            let (meets_gas_requirement, gas_payment) = self
-                .gas_payment_enforcer
-                .message_meets_gas_payment_requirement(&msg.message, &tx_cost_estimate)
-                .await?;
-            if !meets_gas_requirement {
-                info!(gas_payment=?gas_payment, "Gas payment requirement not met yet");
+            Ok(tx_cost_estimate) => tx_cost_estimate,
+            Err(error) => {
+                info!(?error, "Error estimating process costs");
                 return Ok(false);
             }
+        };
 
-            // Go ahead and attempt processing of message to destination chain.
-            debug!(gas_payment=?gas_payment, msg=?msg, "Ready to process message");
+        // If the gas payment requirement hasn't been met, move to the next tick.
+        let (meets_gas_requirement, gas_payment) = self
+            .gas_payment_enforcer
+            .message_meets_gas_payment_requirement(&msg.message, &tx_cost_estimate)
+            .await?;
+        if !meets_gas_requirement {
+            info!(?gas_payment, "Gas payment requirement not met yet");
+            return Ok(false);
+        }
 
-            // TODO: consider differentiating types of processing errors, and pushing to the front of the
-            // run queue for intermittent types of errors that can occur even if a message's processing isn't
-            // reverting, e.g. timeouts or txs being dropped from the mempool. To avoid consistently retrying
-            // only these messages, the number of retries could be considered.
+        // Go ahead and attempt processing of message to destination chain.
+        debug!(gas_payment=?gas_payment, msg=?msg, "Ready to process message");
```

Comment:
message instead of msg? Would be good to be consistent everywhere

Reply:
Actually, isnt' this already in teh span?

Reply:
Yes it is, removed.